ddtrace/tracer/textmap_test.go

```diff
@@ -401,6 +406,9 @@ func TestTextMapPropagator(t *testing.T) {
 	})
 
 	t.Run("InjectExtract", func(t *testing.T) {
+		// TODO: Why is the 128-bit trace id not being propagated correctly by default?
+		os.Setenv("DD_TRACE_128_BIT_TRACEID_GENERATION_ENABLED", "false")
+		defer os.Unsetenv("DD_TRACE_128_BIT_TRACEID_GENERATION_ENABLED")
```

Comment:
I'd like to figure out why this was necessary, and either fix the test or fix the code.